content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield-nodestore.js

```diff
@@ -22,89 +22,89 @@
-                                            sling:resourceType="granite/ui/components/foundation/form/fieldset"
-                                            acs-commons-nested="NODE_STORE"
-                                            name="./pages">
-                                                <layout
-                                                jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/foundation/layouts/fixedcolumns"
-                                                method="absolute"/>
-                                                <items jcr:primaryType="nt:unstructured">
-                                                    <column
-                                                    jcr:primaryType="nt:unstructured"
-                                                    sling:resourceType="granite/ui/components/foundation/container">
-                                                        <items jcr:primaryType="nt:unstructured">
-                                                            <page
-                                                            jcr:primaryType="nt:unstructured"
-                                                            sling:resourceType="granite/ui/components/foundation/form/textfield"
-                                                            fieldDescription="Name of Page"
-                                                            fieldLabel="Page Name"
-                                                            name="./page"/>
-                                                            <path
-                                                            jcr:primaryType="nt:unstructured"
-                                                            sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
-                                                            fieldDescription="Select Path"
-                                                            fieldLabel="Path"
-                                                            name="./path"
-                                                            rootPath="/content"/>
-                                                        </items>
-                                                    </column>
-                                                </items>
-                                            </field>
-                                        </pages>
-                                    </items>
-                                </column>
-                            </items>
-                        </fieldset>
-                    </items>
-                </column>
-            </items>
-        </content>
-    </jcr:root>
-</code>
-*/
+ <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```

Comment:
Can you PLEASE stop including the stopp account on these? I have nothing to
do with Adobe or this project.



On Thu, May 12, 2016 at 2:18 PM, sreek <notifications@github.com> wrote:

> In
> content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield-nodestore.js
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/705#discussion_r63070444>
> :
>
> > -                                                    </column>
> > -                                                </items>
> > -                                            </field>
> > -                                        </pages>
> > -                                    </items>
> > -                                </column>
> > -                            </items>
> > -                        </fieldset>
> > -                    </items>
> > -                </column>
> > -            </items>
> > -        </content>
> > -    </jcr:root>
> > -</code>
> > -*/
> > + <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
>
> comments indentation
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/705/files/b72d57a8f9120ceea632706e336bdde0b7cf4b8d#r63070444>
>



-- 
*Ola Björling* | Global Director of VR
*MediaMonks* | Creative Digital Production
*New York *| Amsterdam | London | Stockholm | Dubai | Los Angeles |
Singapore